config/settings.toml

```diff
@@ -50,3 +50,176 @@ uri = "http://mainnet-router.helium.io:8080/"
 # Maximum number of packets to queue up for the packet router
 queue = 20
 
+# Default target routers for data packets that are not known to helium packet
+# routers. These are legacy settings and will be removed when the Helium Packet
```

Comment:
yes.. the `gateways` and `routers` section are ignored when `validator` is turned off. This will all go away once HPR and the config service is running in a few weeks here 🤞🏻